apps/box/src/navigation/ComponentGallery.navigator.tsx

```diff
@@ -55,6 +56,10 @@ export const ComponentGalleryNavigator = () => {
         name="Avatars"
         component={AvatarDemoScreen}
       />
+      <ComponentGalleryStack.Screen
+        name="Button Groups"
```

Comment:
I generally keep screen names in camel case for navigation etc and then update the title but this works too. 👍